manifests/v/VolkerFischer/Jamulus/3.8.0/VolkerFischer.Jamulus.installer.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: VolkerFischer.Jamulus
+PackageVersion: 3.8.0
+Installers:
+- Architecture: neutral

```

Comment:
```suggestion
- Architecture: x86
```

Reply:
The installer is a shared x86/x64 installer, it installs the right version based on the detected architecture.

Reply:
Changing it to x86 will still run on x64 and arm64 devices.